recipe/meta.yaml

```
@@ -74,12 +74,12 @@ outputs:
         - llvm-openmp  # [osx]
         - libgomp      # [linux]
       host:
-        - nccl         # [linux and cuda_compiler != "None"]
+        - cuda-version {{ cuda_compiler_version }}
+        - nccl                                      # [linux and cuda_compiler != "None"]
```

Good point. Think we want this as well

```suggestion
        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler != "None"]
        - nccl                                      # [linux and cuda_compiler != "None"]
```